examples/Cassie/cassie_rbt_state_estimator.h

```diff
@@ -10,24 +10,111 @@
 #include "systems/framework/output_vector.h"
 #include "systems/framework/timestamped_vector.h"
 #include "examples/Cassie/datatypes/cassie_out_t.h"
+#include "examples/Cassie/cassie_utils.h"
 
 namespace dairlib {
 namespace systems {
 
+using Eigen::Vector3d;
+using Eigen::VectorXd;
+using Eigen::MatrixXd;
+using Eigen::Isometry3d;
+using systems::OutputVector;
+
+using drake::systems::Context;
+using drake::systems::EventStatus;
+using drake::systems::DiscreteValues;
+using drake::systems::DiscreteStateIndex;
+
 /// Translates from a TimestamedVector of cassie torque commands into
 /// a cassie_user_in_t struct for transmission to the real robot.
 class CassieRbtStateEstimator : public drake::systems::LeafSystem<double> {
  public:
-  explicit CassieRbtStateEstimator(
-      const RigidBodyTree<double>&);
+  explicit CassieRbtStateEstimator(const RigidBodyTree<double>&,
+                                   bool is_floating_base);
+  void solveFourbarLinkage(double* left_heel_spring,
+                           double* right_heel_spring,
+                           const VectorXd& q_init) const;
+
  private:
-  void Output(const drake::systems::Context<double>& context,
-      OutputVector<double>* output) const;
+  void AssignImuValueToOutputVector(
```

Comment:
Here and elsewhere. Having the output pointer be the first argument is a little awkward. Would prefer it be the second/last argument.